tokio-epoll-uring/src/ops/fallocate.rs

```diff
@@ -0,0 +1,69 @@
+use std::os::fd::AsRawFd;
+
+use uring_common::{
+    io_fd::IoFd,
+    io_uring::{self},
+};
+
+use crate::system::submission::op_fut::Op;
+
+pub mod mode {
+    pub const ALLOCATE: i32 = 0x00;
+    pub const KEEP_SIZE: i32 = 0x01;
+    pub const PUNCH_HOLE: i32 = 0x02;
+    pub const COLLAPSE_RANGE: i32 = 0x08;
+    pub const ZERO_RANGE: i32 = 0x10;
+    pub const INSERT_RANGE: i32 = 0x20;
+    pub const UNSHARE_RANGE: i32 = 0x40;
+}
```

Comment:
instead of this, re-export 

https://docs.rs/nix/0.30.1/nix/fcntl/struct.FallocateFlags.html

and have the user specify those as `mode` field in `FallocateOp` instead of an `i32`